solidity/random-beacon/contracts/libraries/Groups.sol

```diff
@@ -22,7 +22,8 @@ library Groups {
     struct Group {
         bytes groupPubKey;
         uint256 activationBlockNumber;
-        uint32[] members;
+        // Hash of active group members identifiers array (excluding IA/DQ members).
+        bytes32 activeMembersHash;
```

Comment:
As this is declared in the `Group` struct context this could be `membersHash`
```suggestion
        bytes32 membersHash;
```

Reply:
done